src/i18n/languages/ko_KR.ts

```diff
@@ -0,0 +1,38 @@
+import Key from '../i18nKey'
+import type { Translation } from '../translation'
+
+export const ko_KR: Translation = {
```

Comment:
I think renaming this to `ko` would be better, making it consistent with `en` and `jp`.

Reply:
ok